docassemble/MAVirtualCourt/data/questions/209A_affidavit.yml

```diff
@@ -0,0 +1,499 @@
+question: |
+  % if started:
+    Another incident date
+  % else:
+    Most recent date
+  % endif
+subquestion: |
+  When was the last time ${defendant} abused you?
+  We need to put the year on the form.
+  
+  For the date, try to remember the exact date. If you are not sure of the exact date, do not guess.
+fields:
+  - Year: incidents_of_abuse[i].year
+    code: |
+      reversed(list(range(today().year - 20, today().year + 1)))
+  - Date: incidents_of_abuse[i].date
+  - note: |
+      It is okay to give an approximate date or time, or to put a time of year, like "last summer".
+---
+id: 209A Affidavit time of day
+question: | 
+  Time of day
+subquestion: |
+  What time of day did it happen?
+fields:
+  - no label: incidents_of_abuse[i].time_of_day
+help:
+  label: What if I cannot remember?
+  heading: What if I cannot remember?
+  content: |
+    Do the best you can. Put in the time or describe the time of day.
+    For example, you can say "3:00 p.m.," or "bedtime."
+---
+id: 209A Affidavit Location
+question: |
+  Location
+subquestion: |
+   Where were you? 
+   
+   Be as specific as possible. 
+   
+   For example, "my living room" or "at ______ Park."
+fields:
+  - no label: incidents_of_abuse[i].location
+---
+id: 209A Affidavit Type of violence
+question: |
+   Check a box
+subquestion: |
+   The 209A complaint form has these 4 boxes on it. 
+   
+   We have to check at least one box to get a restraining order.
+   
+   Use the green button to help you decide the boxes you need to check.
+fields:
+  - ${ defendant } made me AFRAID that ${ defendant }  was going to seriously hurt me, physically.: incidents_of_abuse[i].abuse.feared_harm
+    datatype: yesnowide
+  - ${ defendant } tried to physically hurt me.: incidents_of_abuse[i].abuse.defendant_abuse_attempted_physical_harm
+    datatype: yesnowide
+  - ${ defendant } physically hurt me.: incidents_of_abuse[i].abuse.defendant_abuse_caused_physical_harm
+    datatype: yesnowide
+  - ${ defendant }  threatened, pressured or forced me to have sex with ${ defendant }.: incidents_of_abuse[i].abuse.defendant_abuse_caused_sexual_assault
+    datatype: yesnowide
+help:
+  label: Can you give me some examples?
+  heading: Can you give me some examples?
+  content: |
+    Making you afraid of physical harm includes saying things like "I'll kill you if you leave me," or "If you don't give me the car keys, I'll hit you harder than last time." It can also include waving a gun or knife at you, smashing your furniture, driving recklessly, and hurting or killing your pets.It includes threatening to commit any act of violence, threatening with a fake gun, or bomb threats. 
+    
+    slamming fist into hand, into wall, blocking not letting you get by, standing between you and the door, not letting you get back.... 
+    
+    Looming over you, menacing..Spitting on you, punching %%HISHER([Defendant gender MC])%% fist into %%HISHER([Defendant gender MC])%% hand, putting a fist through the wall, 
```

Comment:
Is this formatting other than mako formatting? In my interview this appears like this literally. Is this similar to what you're going for? `${defendants[0].pronoun_possessive('fist')}` ([da docs here](https://docassemble.org/docs/objects.html#how).)